CONTRIBUTORS.md

```diff
@@ -0,0 +1,47 @@
+
+
+# How to contribute
+
+Mesmer is a project to extract and export akka specific metrics.
+
+Projects is comprised of 3 major parts:
+- java agent (called `agent` from now on) - a piece of technology allowing to modify classes during load.
+- akka extension (called `extension` from now on)  - this is where all custom metrics computation happens.
+- application that is being instrumented - in repo example, but this in theory this could be any application that uses akka.
```

Comment:
```suggestion
- application that is being instrumented - in repo we have the "example" app, but this in theory this could be any application that uses akka.
```